package.json

```diff
@@ -69,5 +69,8 @@
     "sass-loader": "^10.1.0",
     "turbo": "^1.2.2",
     "typescript": "^4.0.3"
+  },
+  "resolutions": {
```

Comment:
Added because of [this issue](https://github.com/facebook/react/issues/24304). See [this Slack comment](https://openmrs.slack.com/archives/CHP5QAE5R/p1655485039177609?thread_ts=1655473956.165849&cid=CHP5QAE5R) for details.